cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/PullCommand.java

```diff
@@ -66,11 +66,11 @@ public void execute() {
         }
 
         if (argList == null || argList.size() == 0) {
-            throw LauncherUtils.createUsageExceptionWithHelp("no module given");
+            throw LauncherUtils.createUsageExceptionWithHelp("The Module is not provided.");
```

Comment:
```suggestion
            throw LauncherUtils.createUsageExceptionWithHelp("The module is not provided.");
```

I still prefer the former or even "No module provided." :) IMO, we should look at how others handle these error messages too.

Reply:
The recommendation is to have meaningful and complete sentences as much as possible.